src/main/java/com/revature/repositories/ResultSummaryRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.revature.repositories;
+
+import com.revature.models.ResultSummary;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ResultSummaryRepository extends JpaRepository<ResultSummary, Integer> {
+    
+    ResultSummary findById(int id);
```

Comment:
I believe there is a built in `findById` method already. It might be named slightly differently however.

Reply:
We found it more convenient to have our own findById since the built in one does not directly return ResultSummary, but instead an Optional<ResultSummary>. It also works with our logic much better for our existing service layer when checking for nulls.